scripts/ci/package-unica-plugin.py

```
@@ -431,40 +440,84 @@ def _lower_hex(value: str, length: int) -> bool:
                 f"{data.get('pluginVersion')} != {plugin_version}"
             )
         if data.get("targetTriple") != target_triple:
             raise SystemExit(f"runtime target triple mismatch for {target}")
+
         asset = data.get("asset", {})
-        expected_asset = f"unica-runtime-{target}.tar.gz"
-        if asset.get("name") != expected_asset or asset.get("mediaType") != "application/gzip":
-            raise SystemExit(f"runtime asset identity mismatch for {target}")
+        origin = data.get("assetOrigin")
+        if role == "core":
+            # Ядро собирается здесь, и его имя выводится единым правилом.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF003)